types/leaflet-routing-machine/index.d.ts

```diff
@@ -89,7 +89,7 @@ declare module 'leaflet' {
             waypointNameFallback?: ((latLng: LatLng) => string) | undefined;
             createGeocoder?: ((waypointIndex: number, numberWaypoints: number, plan: Plan) => {}) | undefined;
             addButtonClassName?: string | undefined;
-            createMarker?: ((waypointIndex: number, waypoint: Waypoint, numberWaypoints: number) => Marker) | undefined;
+            createMarker?: ((waypointIndex: number, waypoint: Waypoint, numberWaypoints: number) => Marker | boolean) | undefined;
```

Comment:
Let's update `numberWaypoints` as `numberOfWaypoints`.

**Documentation**
Creates a marker to use for a waypoint. The function should have the signature createMarker(<Number> i, <[L.Routing.Waypoint](https://www.liedman.net/leaflet-routing-machine/api/#l-routing-waypoint)> waypoint, <Number> n), where i is the waypoint’s index, waypoint is the waypoint itself, and n is the total _number of waypoints_ in the plan; if return value is falsy, no marker is added for the waypoint

Reply:
Okay, in the createGeocoder function and others too?

Reply:
If you have time :) Thanks a lot